builds/misc/images-release.yaml

```diff
@@ -1,6 +1,3 @@
-trigger: none
-pr: none
-
```

Comment:
We need this here or else the release pipeline will be triggered by our commits.

Reply:
Added